ucontrib/CoreCrypto/fst/CoreCrypto.fst

```diff
@@ -1,6 +1,6 @@
 module CoreCrypto
 
-open Platform.Bytes
+open FStar.Bytes
 
```

Comment:
Sadly, this change a mistake (a confusion between `ucontrib` and `contrib`) and reverted.
Other people may be using the `ucontrib` file, so I'm not sure we can delete it.

Reply:
Yes, I undid this one because it triggers failures in the examples. They are probably trivial to fix, but miTLS doesn't use ucontrib at all anymore, so from my point of view we can remove it.